apps/dotcdn/src/app/app.enums.ts

```diff
@@ -0,0 +1,16 @@
+export const enum ChartPeriod {
```

Comment:
Why we don't just group `app.enums.ts` and `app.interface.ts` in one file `app.models.ts`?